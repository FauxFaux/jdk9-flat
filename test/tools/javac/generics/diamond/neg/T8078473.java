/*
 * @test /nodynamiccopyright/
 * @bug 8078473
 * @summary  javac diamond finder crashes when used to build java.base module
 * @compile/ref=T8078473.out T8078473.java -XDrawDiagnostics -XDfind=diamond
 */

class T8078473<P, Q> {

    static class C<T, U> {
        C(T8078473<?, ?> p) {}
    }

    {
        new C<Q, Q>(this) {};
        new C<Q, Q>(this);
    }
}
