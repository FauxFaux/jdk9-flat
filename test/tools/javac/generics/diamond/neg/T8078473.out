T8078473.java:15:14: compiler.warn.diamond.redundant.args.1: T8078473.C<Q,Q>, T8078473.C<java.lang.Object,java.lang.Object>
T8078473.java:16:14: compiler.warn.diamond.redundant.args.1: T8078473.C<Q,Q>, T8078473.C<java.lang.Object,java.lang.Object>
2 warnings
